workergroups.tf

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022, Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+module "workergroup" {
```

Comment:
It looks like this module is being invoked without any control or dependencies e.g. it should at least wait for the cluster to be ready.

I'm not sure this is the right level to place this module either. Happy to discuss.

Reply:
Referencing variables from a module establishes dependencies implicitly, no?